src/KeyDB.php

```diff
@@ -39,17 +40,17 @@ class KeyDB
      * @return \Tuf\KeyDB
      *     The constructed key database object.
      *
-     * @throws \Exception
-     *   Thrown if an unsupported key type exists in the metadata.
+     * @throws \Tuf\Exception\InvalidKeyException
+     *   Thrown if an unsupported or invalid key exists in the metadata.
```

Comment:
Why does the method comment include a `@throws` but the method does not throw an exception?

Reply:
I think this ok because `addKey()` is private(for now) so a caller of the public `createFromRootMetadata()` should know the exception could be thrown.